sycl/doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc

```diff
@@ -642,55 +642,70 @@ Automated mechanisms are possible to provide uniquification across calls, and co
 ** `C` is an integer: The relative clock cycle difference between the target anchor and the current function call, that the constraint should infer subject to the type of the control (exact, max, min).
 
-The template arguments above don't have to be specified if user doesn't want to apply latency controls. The template arguments can be passed in arbitrary order.
-
-=== Implementation
+=== Synopsis
 
 [source,c++]
 ----
 // Added in version 2 of this extension.
-namespace sycl::ext::intel::experimental {
-enum class type {
-  none, // default
-  exact,
-  max,
-  min
-};
-
-template <int32_t _N> struct latency_anchor_id {
-  static constexpr int32_t value = _N;
-  static constexpr int32_t default_value = -1;
-};
-
-template <int32_t _N1, type _N2, int32_t _N3> struct latency_constraint {
-  static constexpr std::tuple<int32_t, type, int32_t> value = {_N1, _N2, _N3};
-  static constexpr std::tuple<int32_t, type, int32_t> default_value = {
-      0, type::none, 0};
-};
+#include <sycl/ext/oneapi/latency_control/properties.hpp>
+#include <sycl/ext/oneapi/properties/properties.hpp>
+//***********sycl/ext/oneapi/latency_control/properties.hpp***************
+// enum class latency_control_type {
+//   none, // default
+//   exact,
+//   max,
+//   min
+// };
+//
+// struct latency_anchor_id_key {
+//   template <int Anchor>
+//   using value_t = property_value<latency_anchor_id_key,
+//                                  std::integral_constant<int, Anchor>>;
+// };
+//
+// struct latency_constraint_key {
+//   template <int Target, latency_control_type Type, int Cycle>
+//   using value_t =
+//       property_value<latency_constraint_key,
+//                      std::integral_constant<int, Target>,
+//                      std::integral_constant<latency_control_type, Type>,
+//                      std::integral_constant<int, Cycle>>;
+// };
+//************************************************************************
 
 template <typename name,
           typename dataT,
           size_t min_capacity = 0>
 class pipe {
   // Blocking
-  template <class... _Params>
   static dataT read();
-  template <class... _Params>
-  static void write( const dataT &data );
+
+  template <typename _propertiesT>
+  static dataT read( _propertiesT Properties );
+
+  static void write( const dataT &data);
+
+  template <typename _propertiesT>
+  static void write( const dataT &data, _propertiesT Properties );
 
   // Non-blocking
-  template <class... _Params>
   static dataT read( bool &success_code );
-  template <class... _Params>
+
+  template <typename _propertiesT>
+  static dataT read( bool &success_code, _propertiesT Properties );
+
   static void write( const dataT &data, bool &success_code );
+
+  template <typename _propertiesT>
+  static void write( const dataT &data, bool &success_code, _propertiesT Properties );
 }
 } // namespace sycl::ext::intel::experimental
```

Comment:
Note that this namespace isn't defined above.  Add a namespace declaration at the top of the synopsis, so the reader knows what namespace contains all these definitions.  Also, I presume this should be `sycl::ext::oneapi::experimental` not `sycl::ext::intel::experimental`, right?

Note that the example below also uses the `intel` namespace when I presume you mean the `oneapi` one.

Reply:
I will add a namespace declaration at the top.
It is `sycl::ext::intel::experimental`, where formal/experimental pipe/LSU are defined.

Reply:
Since the property list is already in `sycl::ext::oneapi::experimental` namespace, we cannot avoid having two namespaces here anyway. 
I was trying to at least make the new properties consistent with the property list.

Reply:
Sure. I will move my new properties to the `sycl::ext::intel::experimental` namespace.